### ISSUE TITLE
Concurrency for `ivar trim`?

### ISSUE THREAD

**Issue by philiptzou** [andersen-lab/ivar#139]
Hello, I'm wondering if you have plan to support multi-threads/workers in `ivar`?

**Comment by cmaceves** [comment]
Hi! Not the immediate future. Is there a specific case or dataset you have where you feel that would be vital? (ie. are you running into issues where process time is too long in the current implementation)

**Comment by philiptzou** [comment]
@cmaceves Thanks for the quick response. No I didn't run into any issues - but just discovered iVar is the slow bottleneck in our pipeline and I found the CPU usage of iVar never exceeds 100% (`top -c`).

I can imagine it is in theory practical to use multiple CPUs to process a BAM file includes short reads, since you can just make chunks and deliver the chunks to sub-workers. That's the reason why I ask if you have plan to do so.

**Comment by cmaceves** [comment]
@philiptzou Yeah - I agree that definitely seems within reason and practical as a feature request for future implementation. We're getting ready for a release in the next month or so, but will keep this in mind for future milestones. Thanks for brining this to our attention!

**Comment by philiptzou** [comment]
@cmaceves Thank you!

**Comment by khyox** [comment]
> (...) We're getting ready for a release in the next month or so, but will keep this in mind for future milestones. (...)

Hi @cmaceves. Are you still planning an imminent new release? Thanks!

**Comment by cmaceves** [comment]
@khyox hey! I'm caught up in doctorate applications at the moment, we're anticipating a release in December before the holiday season

**Comment by khyox** [comment]
Great news!! Thank you very much @cmaceves and best of luck with the applications! :)

**Comment by khyox** [comment]
Hey @cmaceves! Hope you were lucky with your PhD applications. 
I guess the release plans got delayed somehow. Are you still planning a new version soon? Thanks!

**Comment by cmaceves** [comment]
hi @khyox  thank you, it went well. we're in the process of doing some final code reviews prior to release, it's still imminent!

**Comment by khyox** [comment]
Both great news! :) Thank you so much @cmaceves!
